presto-raptor/src/main/java/com/facebook/presto/raptor/storage/StorageManagerConfig.java

```diff
@@ -173,6 +174,12 @@ public boolean isOrcLazyReadSmallRanges()
         return orcLazyReadSmallRanges;
     }
 
+    @NotNull
+    public boolean isZstdJniDecompressionEnabled()
```

Comment:
We usually put getter and setting next to each other. Check how other configs are placed.